### ISSUE TITLE
Bug Report / Violation of Design Principles

### ISSUE THREAD

**Issue by IamaUU** [Audiveris/audiveris#926]
1. Violation of Universal Accessibility for Constants: A declared design goal of Audiveris is to make all engine constants accessible and settable by the user. No access for ui constants relating to the right-hand pane. Enabling the Pixel Board (or any of the seven nondefault options) is lost upon moving to a new page. The UI resets to a default state, creating a repetitive manual burden when auditing a large project.

Trivial to reproduce:
- Enable Pixel board
- Select a new page
- Pixel board is gone and must be turned on again.

2. Also documented logical constants (e.g., org.audiveris.omr.sig.ClefHandler.maxDistance) are no longer visibly present in the Tools > Constants dialog. If that example is deleted completely it is a regression in function. If it is still there internally, the option is no longer accessible

Proposed solution
Additions to Tools->Constants
- for all seven nondefault options for tight hand window
- for ClefHandler.maxDistance (if it still exists internally)

My version:
Audiveris
Role: Optical Music Recognition
Version: 5.10.2
Commit: 1b7cf44088c68f4168801822a613751d1bb1b584
Container: file:/C:/Program%20Files/Audiveris/app/audiveris.jar
Home page: http://www.audiveris.org
Project: https://github.com/Audiveris/audiveris
Software license: GNU Affero GPL v3
OCR engine: Tesseract OCR, version 5.5.1
OCR folder: C:\Users\sbhol\AppData\Roaming\AudiverisLtd\audiveris\config\tessdata
Installed languages: deu,eng,fra,ita
Java vendor: Azul Systems, Inc.
Java version: 25.0.2
Java runtime: OpenJDK Runtime Environment (build 25.0.2+10-LTS)
Java VM: OpenJDK 64-Bit Server VM (build 25.0.2+10-LTS, mixed mode)
Operating System: Windows 11 10.0
Architecture: amd64

**Comment by hbitteur** [comment]
> 1. Enabling the Pixel Board (or any of the seven nondefault options) is lost upon moving to a new page.

I don't fully agree: The Pixel board is not **_lost_** upon moving to a new sheet. If you go back to a previous sheet, you can see that its Pixel board is still there!

My understanding is that you expected the selection of non-default boards to apply to **all** sheets. Past, current and future.
While in reality the selection of non-default board is linked to the current underlying sheet.

The strategy is to limit the default boards to the most relevant ones for the tab at hand. For instance, the Pixel board is very relevant for the Gray and Binary tabs, and less relevant for the Data tab. Knowing that the user is always able to modify the boards selection for the sheet and tab at hand.

I don't think it would make sense to pre-select say the "Section Horizontal" board for all sheets of a 100-sheet book.

**Comment by hbitteur** [comment]
> 2\. Also documented logical constants (e.g., org.audiveris.omr.sig.ClefHandler.maxDistance) are no longer visibly present in the Tools > Constants dialog.

I can't find any "ClefHandler" in Audiveris source code or handbook.
Could you double-check the name?

**Comment by hbitteur** [comment]
> 1. Violation of Universal Accessibility for Constants: A declared design goal of Audiveris is to make all engine constants accessible and settable by the user. No access for ui constants relating to the right-hand pane. Enabling the Pixel Board (or any of the seven nondefault options) is lost upon moving to a new page. The UI resets to a default state, creating a repetitive manual burden when auditing a large project.

To address your comment about the "Violation of Universal Accessibility for Constants", please refer to the `Tools | Constants` dialog box. Locate the "selectPixelBoard" constant. By default, its value is "false". You can manually set it to "true", and there you have it! The PixelBoard will now be selected by default!

There, you can also pre-select or pre-deselect any available board defined for the Sheet Editor (the editor accessible via the Data tab): Classifier, Glyph, HorizontalSection, Inter, **Pixel**, Shape, VerticalSection.
